_pages/en_US/credits.md

```diff
@@ -9,20 +9,20 @@ This is the credits for those who have helped out with the guide website, homebr
 If you like what they have done, consider donating (if they have a donation link).
 
 {% capture notice-1 %}
-- [NightScript](https://nightyoshi370.github.io/), [Pk11](https://pk11.us/) & [urmum_69](https://github.com/urmum-69): Guide Maintainers
-- [YourKalamity](https://github.com/YourKalamity/): Making Lazy DSi File Downloader
+- [NightScript](https://nightyoshi370.github.io/), [Pk11](https://pk11.us/) & [urmum_69](https://github.com/urmum-69): Guide maintainers
+- [YourKalamity](https://github.com/YourKalamity/): Creator of Lazy DSi File Downloader
 - [emiyl](https://emiyl.com/paypal): Domain holder
 - Halvorsen: Provided an image of their Nintendo DSi to be used on the home page
 - [shutterbug2000](https://paypal.me/projectkaeru): Discoverer & Developer of DSi Camera & Flipnote Studio exploits
-- [WinterMute](https://devkitpro.org/support-devkitpro): Maintainer of the DS Homebrew Tools & forked Flipnote Studio exploit to make Flipnote Lenny (supports all firmwares and regions)
-- [nocash](https://www.patreon.com/martin_korth): Unlaunch Developer
+- [WinterMute](https://devkitpro.org/support-devkitpro): Maintainer of the DS Homebrew Tools & forked the ugopwn Flipnote Studio exploit to make Flipnote Lenny (supports all firmwares and regions)
```

Comment:
It might be worthwhile to state that this is talking about the development toolchain

Reply:
How is it now?